pkgs/http/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.5.0
```

Comment:
You can move this into the 1.4.0-wip section.

And remove "with fixing tests".

Reply:
Ok